README.rst

```diff
@@ -54,6 +54,9 @@ Deploy the downscaler into your cluster via (also works with kind_ or Minikube_)
 
     $ kubectl apply -f deploy/
 
+In case you are deploying `kube-downscaler` to another namespace than `default`, for example if your context is pointing to `my-namespace`, make
```

Comment:
It's a reStructuredText file (not Markdown), so you need to use two backticks for literals.